tests/counterpoll_input/config_db.json

```diff
@@ -1,2668 +1,2671 @@
 {
```

Comment:
I believe there are so much white space removal from this file which makes it harder to review the diff. Can you please make sure only functional changes are present and revert it as previous.
We can raise the indentation and whitespace removal as a separate PR if wanted.